app/src/main/res/layout/item_time_record.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Hoàn toàn có thể dùng backgroundDrawable cho constraintlayout bên dưới mà. Không nên phân cấp 2 parent thế này đâu ô, hiệu năng sẽ giảm, FrameLayout trong trường hợp này chẳng có tác dụng gì ngoài việc bo cái góc mà drawable cũng làm được.